src/app/personside/Personside.tsx

```diff
@@ -33,14 +35,26 @@ class Personside extends React.PureComponent<PersonsideStateProps> {
         super(props);
     }
 
+    visInnhold() {
+        if (instanceofBegrensetInnsyn(this.props.personReducer.data as PersonRespons)) {
```

Comment:
Jeg synes fremdeles det ser tungvint ut å drive å caste denne responsen overalt. 

Her castes det rett til PersonResponser. Vil ikke det kræsje hvis responsen ikke er instance of PersonResponse?